content/docs/quickstart.mdx

```diff
@@ -51,7 +51,7 @@ Add the configuration below to `docker-compose.yaml`:
 
 <DockerCompose />
 
-## Run Docker Compose
+**Run Docker Compose:**
```

Comment:
What's this change for?

Reply:
I may have done that by accident when resolving a merge conflict. It should be a an H2.